OpenXmlPowerTools/HtmlToWmlConverterCore.cs

```diff
@@ -2829,9 +2829,9 @@ private static XElement[] GetSpacingProperties(XElement paragraph, HtmlToWmlConv
             // todo should check based on display property
             bool numItem = paragraph.Name == XhtmlNoNamespace.li;
 
-            if (numItem && marginTopProperty.IsAuto)
+            if (numItem && marginTopProperty != null && marginTopProperty.IsAuto)
```

Comment:
Why not just use
`if (numItem && marginTopProperty?.IsAuto)'

Reply:
The reason is that numItem is 'bool', but marginTopProperty?.IsAuto is 'bool?' , so '&&' operator cannot be applied to 'bool' and 'bool?'.

Thank you for your review!

Reply:
if (numItem && marginTopProperty?.IsAuto == true)